R/geom-smooth.r

```diff
@@ -20,14 +20,14 @@
 #' @seealso See individual modelling functions for more details:
 #'   [lm()] for linear smooths,
 #'   [glm()] for generalised linear smooths,
```

Comment:
If we're giving this one sentence structure (which we are, since there's a period after the last one) there should be an "and" added to the second to last list item. (Basically, "bulleted lists within sentences" structure: http://blog.apastyle.org/apastyle/2010/03/lists-part-5-bulleted-lists.html)